js/capitalize.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(str) {
```

Comment:
I moved this away from prototype modification, because usually touching the prototypes of built-ins in JavaScript is a bad idea. That said, I've modified both Function and Array prototypes in the past, and I probably will again, so do as I say and not as I do?